src/magento-payments/payments.md

```diff
@@ -1,3 +1,84 @@
 ---
 title: Magento Payments
----
\ No newline at end of file
+---
+
+Magento Payments is your turnkey self-service solution, including sandbox testing and a super simple setup, for providing robust and secure payment processing for your Magento websites.
+
+Whether you are a small business, mid-market contender, or large enterprise, our white-labeled pre-installed payments and risk solution will help you reduce operational overhead while increasing revenue, driving greater merchant adoption rates, and vastly improving the user's checkout experience.
```

Comment:
Is "white-labeled" correct? Isn't it labelled as "Magento Payments"?

Reply:
This was from previous text/copy. I think you're right. Changing.